pipelines/README.md

```diff
@@ -102,13 +102,41 @@ oc secret link pipeline rhoai-edge-build-pull-secret
 
 ### Deploy and run the test pipeline
 
-> **NOTE**
-> Make sure to change the `target-imagerepo` parameter to match the name of your Quay namespace if using one of the provided `PipelineRun` files.
+Update the `target-imagerepo` parameter value from its default `rhoai-edge` in
+[`test-mlflow-image-pipelinerun-tensorflow-housing.yaml`](tekton/test-mlflow-image-pipeline/test-mlflow-image-pipelinerun-tensorflow-housing.yaml)
+and/or
+[`test-mlflow-image-pipelinerun-bike-rental.yaml`](tekton/test-mlflow-image-pipeline/test-mlflow-image-pipelinerun-bike-rental.yaml)
+to match the name of your Quay namespace.
+
+Then create the pipeline(s) to test the container images behaviour with
```

Comment:
```suggestion
Then create the pipeline(s) to test the container images behaviour using
```

Reply:
Thanks, updated in 69c0aa71394c8642c8fb6a1ce0f278ae019605b7.